### DISCUSSION TITLE
Defectdojo and Nessus Integration

### DISCUSSION THREAD

**Discussion by Muhammad-Irtaza** [DefectDojo/django-DefectDojo#8595] [Q&A] [answered]
Hi,
I need help, how can I automate this: integrate nessus scan results and directly import to defectdojo.
Your help will be really helpful.
TIA

**Comment by manuel-sommer** [comment, marked as answer]
Hi @Muhammad-Irtaza 

You have to use the Tenable Parser https://defectdojo.github.io/django-DefectDojo/integrations/parsers/file/tenable/ for this. Furthermore, for automation you can use the REST API to push the findings to DefectDojo.

Best,

Manuel

**Reply by nour1509** [reply]
Hello i need to ask you how we can use the REST API to push findings to DefectDojo from nessus

**Reply by manuel-sommer** [reply]
Build an automation and use scripts. I don't have access to nessus.
